services/storage/object/capabilities.yaml

```diff
@@ -1,85 +1,99 @@
-common_features:
-  - CCC.F01 # Encryption in Transit Enabled by Default
-  - CCC.F02 # Encryption at Rest Enabled by Default
-  - CCC.F03 # Access/Activity Logs
-  - CCC.F04 # Transaction Rate Limits
-  - CCC.F05 # Signed URLs
-  - CCC.F06 # Identity Based Access Control
-  - CCC.F07 # Event Notifications
-  - CCC.F08 # Multi-zone Deployment
-  - CCC.F09 # Monitoring
-  - CCC.F10 # Logging
-  - CCC.F11 # Backup
-  - CCC.F12 # Restore
-  - CCC.F14 # API Access
-  - CCC.F18 # Versioning
-  - CCC.F20 # Tagging
-  - CCC.F21 # Replication
-  - CCC.F22 # Location Lock-In
+shared-capabilities:
```

Comment:
Didn't we decide in a previous meeting to not rename features to capabilities?

Reply:
Would like to see mappings for controls at least moved into another file.   There are various pros and cons of this:

1.  Mappings will become neglected if we separate them. 
2. On the other hand, it frees up the security team to not have to worry so much about these.
3. We may be able to generate control mappings using tools
4. We may be able to more easily version control mappings on a per-file basis.  e.g. we can have a separate file for each version of a control catalog./